src/pages/documentation/mobile-core/configuration/api-reference.md

```diff
@@ -50,7 +46,7 @@ iOS
 
 <InlineAlert variant="info" slots="text"/>
 
-This API is only available in Android and was added in Android was added in Android Core version 1.7.0.
+This API is only available in Android.
```

Comment:
We could keep this note, This API is only available in Android, starting with version 1.7.0.

Reply:
@emdobrin Isn't this redundant since these docs are for the "Latest Version"?

Reply:
Yeah, since this is from 2.0 onwards, shouldn't this only be a note in the previous versions one?

Reply:
ok, previous versions sounds good, as long as we don't miss it